src/Solution/SingleFileSolution.php

```diff
@@ -38,7 +38,7 @@ public function __construct(string $file)
      */
     public static function fromFile(string $file): self
     {
-        return new self(InTempSolutionMapper::mapFile($file));
```

Comment:
Dropped this InTemp mapper now and we just do it right before executing instead, because this also fucked up docker but I can't remember how now. So I added a `addFile` method to the context/env which then gets written with the solution. Seems cleaner to do it that way anyway.